packages/gcc10.rb

```diff
@@ -69,128 +76,338 @@ def self.build
-        --enable-threads=posix \
-        --disable-werror \
-        --disable-libmpx \
-        --enable-static \
-        --enable-shared \
-        --with-pic \
-        --program-suffix=-#{gcc_version} \
-        --with-arch-64=x86-64 \
-        --with-build-config=bootstrap-lto-lean"
+        system "env CFLAGS=#{@cflags} CXXFLAGS=#{@cxxflags}  \
+          LD_LIBRARY_PATH=#{CREW_LIB_PREFIX} \
+          LIBRARY_PATH=#{CREW_LIB_PREFIX} PATH=#{@path} \
+          ../configure #{CREW_OPTIONS} \
+          --disable-libmpx \
+          --disable-libssp \
+          --disable-multilib \
+          --disable-werror \
+          --enable-cet=auto \
+          --enable-checking=release \
+          --enable-clocale=gnu \
+          --enable-default-pie \
+          --enable-default-ssp \
+          --enable-gnu-indirect-function \
+          --enable-gnu-unique-object \
+          --enable-host-shared \
+          --enable-languages=#{@languages}  \
+          --enable-lto \
+          --enable-plugin \
+          --enable-shared \
+          --enable-static \
+          --enable-threads=posix \
+          --program-suffix=-#{gcc_version} \
+          --with-arch-64=x86-64 \
+          --with-build-config=bootstrap-lto-lean \
+          --with-isl \
+          --with-pic \
+          --with-system-zlib"
       when 'i686'
-        system "../configure  #{CREW_OPTIONS} \
-        --enable-checking=release \
-        --enable-lto \
-        --disable-multilib \
-        --enable-threads=posix \
-        --disable-werror \
-        --disable-libmpx \
-        --enable-static \
-        --enable-shared \
-        --with-pic \
-        --program-suffix=-#{gcc_version} \
-        --with-arch-32=#{ARCH} \
-        --with-build-config=bootstrap-lto-lean"
+        system "env CFLAGS=#{@cflags} CXXFLAGS=#{@cxxflags} \
+          LD_LIBRARY_PATH=#{CREW_LIB_PREFIX} \
+          LIBRARY_PATH=#{CREW_LIB_PREFIX} PATH=#{@path} \
+          ../configure  #{CREW_OPTIONS} \
+          --disable-libmpx \
+          --disable-libssp \
+          --disable-multilib \
+          --disable-werror \
+          --enable-cet=auto \
+          --enable-checking=release \
+          --enable-clocale=gnu \
+          --enable-default-pie \
+          --enable-default-ssp \
+          --enable-gnu-indirect-function \
+          --enable-gnu-unique-object \
+          --enable-host-shared \
+          --enable-languages=#{@languages}  \
+          --enable-lto \
+          --enable-plugin \
+          --enable-shared \
+          --enable-static \
+          --enable-threads=posix \
+          --program-suffix=-#{gcc_version} \
+          --with-arch-32=#{ARCH} \
+          --with-build-config=bootstrap-lto-lean \
+          --with-isl \
+          --with-pic \
+          --with-system-zlib"
       end
-      system 'make'
+      system "env LD_LIBRARY_PATH=#{CREW_LIB_PREFIX} \
+          LIBRARY_PATH=#{CREW_LIB_PREFIX} PATH=#{@path} \
+          make -j4"
```

Comment:
Will `make -j#{CREW_NPROC}` works?

Reply:
`-j#{CREW_NPROC}` is actually built-in! But for some reason I was only effectively getting `-j1` performance. Either a bug or an issue with how my container was seeing this....

Reply:
try `crew const CREW_NPROC` maybe the nproc variable wasn't getting set correctly

Reply:
`CREW_NPROC` does look like it is populated properly. However, it wasn't being pulled in during make for some reason. I plan on reverting this code to the default before it is moved from draft though.